oak/server/rust/oak_loader/src/main.rs

```diff
@@ -72,7 +73,7 @@ pub struct Opt {
         long,
         help = "PEM encoded X.509 TLS root certificate file used to authenticate an external gRPC service."
     )]
-    root_tls_certificate: String,
+    root_tls_certificate: Option<String>,
```

Comment:
Can we make it so that this field (and flag) only actually exist in the debug build? No need to get out of your way, if there is no simple way.

Reply:
I could not find an easy way to completely remove it, but I changed it so that the structopt attribute is only added when `oak_debug` is enabled. This means that the flag is not available to be set without the feature.